openid-federation-1_0.xml

```diff
@@ -761,9 +761,193 @@
+		the Entity Statement be a Subordinate Statement.
+		Verify that its value is syntactially correct,
+		as specified in <xref target="metadata_policy"/>.
+	      </t>
+	      <t>
+		If the <spanx style="verb">metadata_policy_crit</spanx> Claim is present,
+		the Entity Statement be a Subordinate Statement.
+		Each array element in this claim's value
+		MUST be a string representing a Metadata Policy operator
+		that is not defined by this specification
+		and that operator MUST be understood
+		and be able to be processed by the implementation.
+	      </t>
+	      <t>
+		If the <spanx style="verb">constraints</spanx> Claim is present,
+		the Entity Statement be a Subordinate Statement.
+		Verify that its value is syntactically correct,
+		as specified in <xref target="chain_constraints"/>.
+	      </t>
+	      <t>
+		If the <spanx style="verb">trust_marks</spanx> Claim is present,
+		the Entity Statement MUST be an Entity Configuration.
+		For each element of the array that is the Claim value,
+		validate that the
+		<spanx style="verb">trust_mark_type</spanx> member value matches the
+		<spanx style="verb">trust_mark_type</spanx> Claim value in the
+		Trust Mark JWT that is the value of the
+		<spanx style="verb">trust_mark</spanx> member.
+		Validate each Trust Mark as described in
+		<xref target="trust-mark-validation"/>.
+	      </t>
+	      <t>
+		If the <spanx style="verb">trust_mark_issuers</spanx> Claim is present,
+		the Entity Statement MUST be an Entity Configuration.
+		Validate that its Claim value is a JSON object with
+		Trust Mark type identifiers as the member names and
+		arrays of Entity Identifiers as the values.
+	      </t>
+	      <t>
+		If the <spanx style="verb">trust_mark_owners</spanx> Claim is present,
+		the Entity Statement MUST be an Entity Configuration.
+		Validate that its Claim value is a JSON object with
+		Trust Mark type identifiers as the member names and
+		values that are JSON objects containing
+		a <spanx style="verb">sub</spanx> member with a value that is
+		an Entity Identifier and
+		a <spanx style="verb">jwks</spanx> member with a value that is
+		a JSON Web Key Set.
+	      </t>
+	      <t>
+		If the <spanx style="verb">source_endpoint</spanx> Claim is present,
+		the Entity Statement MUST be a Subordinate Statement.
+		Validate that its Claim value is a URL.
+		Implementations MAY also make a fetch call to the URL
+		to validate that this is the fetch endpoint
+		from which the Entity Statement was issued.
+	      </t>
+	      <t>
+		If the <spanx style="verb">trust_anchor</spanx> Claim is present,
+		validate that its value is an Entity Identifier.
+		Implementations MAY retrieve the Entity Configuration for the
+		Entity Identifier and validate that it is a Trust Anchor,
+		and they MAY also validate that it is the Trust Anchor
+		used for the Explicit Registration.
+		This Claim MUST NOT be present in Entity Statements that are not
+		Explicit Registration responses.
+	      </t>
```

Comment:
update 932-939 with this to support possible multiple trust anchors 
If the <spanx style="verb">trust_anchor</spanx> Claim is present,
validate that its value is an Entity Identifier corresponding to one
of the configured Trust Anchors in the validator’s Anchor Set.
Implementations MAY retrieve the Entity Configuration for the
Entity Identifier and validate that it is a Trust Anchor,
and they MAY also validate that it is the Trust Anchor
used for the Explicit Registration.
Validators MAY also accept Explicit Registration responses that omit
the <spanx style="verb">trust_anchor</spanx> Claim if the associated Trust Chain
terminates at any Trust Anchor in the Anchor Set.